apps/catalogue/src/main.js

```diff
@@ -160,6 +160,12 @@ const router = createRouter({
       component: ResourceRedirectView,
       props: true,
     },
+    {
+      name: "AllVariables-details",
+      path: "/all-variables/:resource/:dataset/:name",
+      redirect: { name: "Variables-details" },
+      props: true,
```

Comment:
Couldn't really find where this path is used. When entering manually the redirect works, but it makes me wonder why this is needed.

Reply:
if you look at screenshot of Erik below you can see a 'target variable'. This one points to the All variables table, however, I know that in this data we actually need to query a subclass called 'variables'. However, in the future we might need to first retrieve AllVariables and then see what subclass it actually is. That is wat we do with Resources, see Resource redirect in the same main.js